oshi-core/src/main/java/oshi/hardware/platform/mac/MacCentralProcessor.java

```diff
@@ -182,9 +182,9 @@ public long[] querySystemCpuLoadTicks() {
     public long[] queryCurrentFreq() {
         if (isArmCpu) {
             Map<Integer, Long> physFreqMap = new HashMap<>();
-            getPhysicalProcessors().stream().forEach(p -> physFreqMap.put(p.getPhysicalProcessorNumber(),
+            getPhysicalProcessors().stream().parallel().forEach(p -> physFreqMap.put(p.getPhysicalProcessorNumber(),
```

Comment:
Probably doesn't hurt but we're just parallelizing inserting into a map, and a relatively small number of physical processors (4 to 8 on most modern machines) so likely not needed.